keps/sig-node/1539-hugepages/README.md

```diff
@@ -546,6 +560,67 @@ locality guarantees as a feature of QoS.  In particular, pods in the
     - Test case will be added to cri-tools to be used in CRI runtime' test(CI).
     - here: https://github.com/kubernetes-sigs/cri-tools
 
+## Test Plan for HugePageStorageMediumSize
+
+- Promote existing HugePages E2E tests to conformance
+
+## Production Readiness Review Questionnaire
+### Monitoring requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+An operator could audit pods that use hugepages resources.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine.
+the health of the service?**
+N/A
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+N/A
```

Comment:
How does a cluster admin expect this feature to perform on a cluster? How can they measure its performance? That is the intent of these questions, I think they are applicable.

Reply:
Your questions seem to be related to the hugepages support in general, which is already GA. I'm going to rename 'Production Readiness Review Questionnaire' item to 'Production Readiness Review Questionnaire for HugePageStorageMediumSize' to solve this. Would it work?

Reply:
Yes, per our slack convo this only needs to be targeted to `HugePageStorageMediumSize`.

Reply:
understood. thank you for the explanations.